replay-tests/helloworld/app/views/tags.html

```
@@ -0,0 +1,22 @@
+<html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)